config/yaml/volta.yaml

```diff
@@ -1,4 +1,4 @@
 packages:
  kokkos:
-  variants: +cuda +openmp +volta70 +cuda_lambda +wrapper ^cuda@10.1
+  variants: +cuda +openmp +volta70 +wrapper ^cuda@10.1
```

Comment:
how is this supposed to work. Either we need cuda_lambda because its an older Kokkos ro we need cuda@11.x